src/plugins/mer/merdevicedebugsupport.cpp

```diff
@@ -113,6 +113,12 @@ MerDeviceDebugSupport::MerDeviceDebugSupport(RunControl *runControl)
 
     setUsePortsGatherer(isCppDebugging(), isQmlDebugging());
 
+    auto aspect = runControl->runConfiguration()->aspect<MerRunConfigurationAspect>();
+    if (aspect->isDebugEnvironmentEnabled()) {
```

Comment:
I would prefer using more specific naming for this and all the related identifiers, e.g. "isDebugBypassOpenSslArmCapEnabled".

Reply:
I agree - for some reason it didn't occur to me to change the names when removing the generic debug environment functionality

Reply:
hm, I still see some occurrences of the original name (setter, member variable, constant IIRC)

Reply:
:( sorry about that, I'll fix them asap